srcpkgs/zabbix/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'zabbix'
 pkgname=zabbix
-version=4.2.1
+version=4.2.2
```

Comment:
It's the revision that needs to be increased, not the version. So please revert that and change the line below to `revision=2`. After that, please squash everything into one commit.